plugins/inputs/exec/README.md

```diff
@@ -11,21 +27,44 @@ and strings will be ignored.
 ```
 # Read flattened metrics from one or more commands that output JSON to stdout
 [[inputs.exec]]
-  # the command to run
-  command = "/usr/bin/mycollector --foo=bar"
+  # Shell/commands array
+  commands = ["/tmp/test.sh","/tmp/test2.sh"]
```

Comment:
This is a breaking change. You'll need to change unit tests for this. Also `exec` should continue to support users that have the `command = ` specified for backwards compatibility.